### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by Districh-ru**
Translate recently changed English phrases.

### CODE REVIEW

**Comment by Branikolog** [comment]
Hi, @Districh-ru !
Can I ask you to fix another artifact description here, please?
image
This artifact adds one additional shot for catapult. This description doesn't fit in a case hero has expert "ballistics skill" when there're already two shots. This artifact adds one more, but not doubles each shot.
So I propose: "Баллиста позволяет произвести один дополнительный выстрел из катапульты за ход."

**Review by Branikolog — APPROVED** [review]

**Comment by Districh-ru** [comment]
> Hi, @Districh-ru !
> Can I ask you to fix another artifact description here, please?


Hi, @Branikolog.
Sure, I'll do it later today.

**Review by ihhub — APPROVED** [review]

File: files/lang/be.po [hunk 1]
```diff
@@ -7685,7 +7685,7 @@ msgstr ""
 msgid "Ballista of Quickness"
 msgstr ""
 
-msgid "The %{name} lets your catapult fire twice per combat round."
+msgid "The %{name} lets your catapult do one extra fire per combat round."
```
[the remark this answers is not in the document]
Comment (Districh-ru) [reply files/lang/be.po]: Thanks, @zenseii!
This sounds much better.

**Comment by Districh-ru** [comment]
@Branikolog, in last commit I added the translation for recently added Virtual Keyboard button info text. Could you please re-review this PR when you have time. :)

File: files/lang/ru.po [hunk 2]
```diff
@@ -3101,15 +3101,14 @@ msgstr "Выбор разрешения экрана:"
...
 msgstr "Щёлкните, чтобы применить выбранное разрешение экрана."
 
-#, fuzzy
 msgid "Click to apply the entered text."
-msgstr "Щёлкните, чтобы применить выбранное разрешение экрана."
+msgstr "Щёлкните, чтобы принять введённый текст."
 
 msgid "Click to open the Virtual Keyboard dialog."
-msgstr ""
+msgstr "Щёлкните, чтобы открыть окно виртуальной клавиатуры."
```
Comment (Branikolog) [inline files/lang/ru.po:3108]: Hi, @Districh-ru !
How do you think about changing "Щёлкните" to more formal "Нажмите"?

Comment (Districh-ru) [reply files/lang/ru.po]: Hi @Branikolog,
I agree with your proposal and applied it to this and other two phrases in the translation file.

**Review by Branikolog — APPROVED** [review]

**Comment by ihhub** [comment]
@Districh-ru , thank you very much for the update!

**Pull request merged**